lib/build/multi.js

```diff
@@ -109,21 +109,34 @@ module.exports = function(config, options){
 				}
 			);
 
-			writeStream.on("data", function(data){
-				this.end();
-
-				// If bundleAssets is truthy run the bundler after the build.
-				if(options && options.bundleAssets) {
-					require("steal-bundler")(data, options.bundleAssets)
-						.then(function(){
-							resolve(data);
-						}, function(err){
-							reject(err);
+			writeStream.on("data", function(builtResult){
+				//this.end();
```

Comment:
@matthewp dont know whats that function is doing. if i comment it out, it worked for the multibuild, but not for the slim build https://github.com/stealjs/steal-tools/pull/848/files#diff-b016b898f98b8eb1dc4d58c367f1bc38R99
is `this.end()` necessary?